ui/src/app/cohort-search/constant.ts

```diff
@@ -37,7 +37,8 @@ export const LIST_PROGRAM_TYPES = [];
 
 export const LIST_DOMAIN_TYPES = [
   {name: 'Measurements', domain: DomainType.MEASUREMENT, type: null},
-  {name: 'Conditions', domain: DomainType.CONDITION, type: null}
+  {name: 'Conditions', domain: DomainType.CONDITION, type: null},
+  {name: 'Drugs', domain: DomainType.DRUG, type: null}
```

Comment:
if type is null for each do we need init that prop?

Reply:
We may need it eventually, but I removed it for now. Will be easy to put back if needed.